core/src/main/java/org/springframework/security/authentication/ObservationAuthenticationManager.java

```diff
@@ -56,4 +57,11 @@ public Authentication authenticate(Authentication authentication) throws Authent
 		});
 	}
 
+	public ObservationConvention<AuthenticationObservationContext> getConvention() {
+		return this.convention;
+	}
+
+	public void setConvention(ObservationConvention<AuthenticationObservationContext> convention) {
```

Comment:
Will you please add a quick JavaDoc here that at least marks the version when it was added? For example:

```java
/**
 * Use this convention for reporting observation data
 * 
 * @since 6.1
 */
```

Reply:
Please also do this for the other setters.